<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slm_control</class>
 <widget class="QMainWindow" name="slm_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="grbSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QPushButton" name="btnLoad">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnSaveSettings">
           <property name="toolTip">
            <string>Save configuration file</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnRegenerate">
           <property name="text">
            <string>Regenerate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="slm_pattern_generation_selector" name="slm_pattern_options" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbAcquisition">
        <property name="title">
         <string>Acquisition Settings</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="chkDisplayacquisition">
           <property name="toolTip">
            <string>Shows the image being acquired. This may slow down highspeed acquisition.</string>
           </property>
           <property name="text">
            <string>Display acquisition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkDisplayProgress">
           <property name="toolTip">
            <string>Write text to console while image are acquired. This may slow down highspeed acquisition.</string>
           </property>
           <property name="text">
            <string>Display Progress</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grb_scope_compute">
        <property name="title">
         <string>Compute Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="scope_compute_settings_selector" name="scope_compute_settings" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grb_dpm">
        <property name="title">
         <string>Off-Axis Band Selection</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="dpm_settings_selector" name="dpm_settings" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbPhaseChannel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Phase Channel</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkTransmission">
           <property name="text">
            <string>Transmission</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Shutter Threshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="qsbTimeout">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="suffix">
              <string> ms</string>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="cmbChannel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GrbLevelMatching">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Calibration Sequences</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QPushButton" name="btnTakeOneRaw">
           <property name="text">
            <string>Take One</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnGLIMShear">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Phase ramp calibration</string>
           </property>
           <property name="text">
            <string>Shear</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnGrayLevelMatching">
           <property name="text">
            <string>Gray Levels</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Patterns</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnSaveFiles">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Save patterns to a folder</string>
           </property>
           <property name="text">
            <string>Save Patterns</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="qsbPatternNumber">
           <property name="toolTip">
            <string>Selected Pattern</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="fixed_modulator_settings_selector" name="fixed_modulator_settings" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>slm_pattern_generation_selector</class>
   <extends>QWidget</extends>
   <header>slm_pattern_generation_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scope_compute_settings_selector</class>
   <extends>QWidget</extends>
   <header>scope_compute_settings_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>fixed_modulator_settings_selector</class>
   <extends>QWidget</extends>
   <header>fixed_modulator_settings_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>dpm_settings_selector</class>
   <extends>QWidget</extends>
   <header>dpm_settings_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbChannel</tabstop>
  <tabstop>chkTransmission</tabstop>
  <tabstop>chkDisplayacquisition</tabstop>
  <tabstop>chkDisplayProgress</tabstop>
 </tabstops>
 <resources>
  <include location="slim5.qrc"/>
 </resources>
 <connections/>
</ui>
